workspace.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {}
+}
```

Comment:
This should not be included.

Reply:
Yes my bad. It keeps getting through the gitignore